### PR TITLE
Improve accessibility for the files sidebar

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
* Transform file name to `h2` element
* Transform `Open folder` action to link instead of button
* Add a more descriptive close button label
* Close sidebar when clicking outside of it
* Remove `aria-label` on the action buttons as they already include proper labels
* Transform the favorite-star in sidebar to a button-element, adjust `aria-label`
* Add a hint for screen readers if an action opens a new window/tab

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicDifferentRoles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinRestore</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>

















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>





























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>





















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>







































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>





































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14963/

<p>









































































































</p></details>

File: packages/web-runtime/src/App.vue [hunk 1]
```diff
@@ -24,7 +24,7 @@
             v-touch:swipe.left="handleNavSwipe"
             class="oc-app-navigation"
             :logo-img="logoImage"
-            :product-name="productName"
+            product-name="ffff"
```
Comment (pascalwengerter) [inline packages/web-runtime/src/App.vue:27]: Was this meant to make it into the commit? ;)

File: packages/web-app-files/src/components/Sidebar.vue [hunk 2]
```diff
@@ -217,4 +219,7 @@ export default {
     fill: #ffba0a !important;
   }
 }
+#files-sidebar-item-name {
+  font-size: medium;
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/Sidebar.vue:223]: While I like the readability of this I'd suggest we stick with either `rem` or our custom css properties (`--oc-font-size-default` in this case I guess) for consistancy?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinFilesFolders</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicDifferentRoles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinRestore</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>

















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>





























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>





















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>





































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIRename</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14968/

<p>









































































































</p></details>

File: packages/web-app-files/src/components/Sidebar.vue (hunk 2 above)
Comment (fschade) [reply packages/web-app-files/src/components/Sidebar.vue]: done 🥳

File: packages/web-runtime/src/App.vue (hunk 1 above)
Comment (fschade) [reply packages/web-runtime/src/App.vue]: done 💯

**Comment by JammingBen** [comment]
Ermmm... @fschade I think you've overwritten my last 2-3 commits including quite some changes :see_no_evil:

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14982/

<p>







</p></details>

**Comment by fschade** [comment]
> Ermmm... @fschade I think you've overwritten my last 2-3 commits including quite some changes 🙈

reverted, sorry

File: packages/web-app-files/src/components/Sidebar.vue [hunk 3]
```diff
@@ -218,4 +222,7 @@ export default {
     fill: #ffba0a !important;
   }
 }
+#files-sidebar-item-name {
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/Sidebar.vue:225]: Can be solved by giving `class="oc-text-initial"` to the h3 🤓

Comment (JammingBen) [reply packages/web-app-files/src/components/Sidebar.vue:225]: Niceeee, was exactly looking for something like this! Thanks 👍

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinFilesFolders</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicDifferentRoles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinRestore</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>

















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>





























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>





















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>







































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15003/

<p>





































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinFilesFolders</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootEdgeCases</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>RestrictSharing</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsEdgeCases</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicManagement</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFilePermissionsGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinRestore</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>

















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderPermissionsGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicDifferentRoles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFilePermissionMultipleUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderPermissionMultipleUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>





























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>





















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Favorites</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>

























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>

























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersExpire</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPermissionsUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>

























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersExpireToRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>





















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderAdvPermissionsGrp</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15012/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinDelete</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>TrashbinFilesFolders</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsEdgeCases</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>RestrictSharing</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>MarkdownEditor</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicManagement</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15015/

<p>





</p></details>

File: packages/web-app-files/src/components/Sidebar.vue [hunk 4]
```diff
@@ -11,25 +13,28 @@
       </div>
       <div class="uk-inline">
         <div class="uk-flex uk-flex-middle">
-          <span
+          <h3
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/Sidebar.vue:16]: @kulmann this should be `<h2>` according to the accessibility call, right?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootEdgeCases</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsEdgeCases</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicExpire</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>









</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicManagement</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>RestrictSharing</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicDifferentRoles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>











</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFilePermissionsGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>













</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderPermissionsGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFilePermissionMultipleUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderPermissionMultipleUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsersRoot1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroups</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





















</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>

























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPermissionsUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>

























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalUsersExpireToRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderAdvPermissionsGrp</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingFolderAdvancedPermissionMU</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPublicBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>



























</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingPermissionToRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>





































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>ResharingToRoot</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>







































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIResharing</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>







































</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15020/

<p>

























</p></details>

File: packages/web-app-files/src/components/Sidebar.vue (hunk 4 above)
Comment (JammingBen) [reply packages/web-app-files/src/components/Sidebar.vue:16]: Makes sense IMO, I'll go for it for now. We can still revert it later if `h3` was correct.

File: packages/web-app-files/src/components/Sidebar.vue [hunk 5]
```diff
@@ -202,6 +207,22 @@ export default {
...
+       */
+
+      if (
+        document.querySelector('.files-topbar').contains(event.target) ||
+        document.querySelector('.oc-topbar').contains(event.target) ||
+        document.querySelector('.oc-app-navigation').contains(event.target) ||
+        event.target.id === 'files-view'
+      ) {
+        this.close()
+      }
```
Comment (JanAckermann) [inline packages/web-app-files/src/components/Sidebar.vue:225]: @pascalwengerter ugly but I don't see another solution here.

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15025/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15030/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15034/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15034/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles1</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15035/

<p>







</p></details>

**Comment by JammingBen** [comment]
Currently fixing the last few tests, I think we can already mark this as ready to review though.

File: packages/web-app-files/src/components/Sidebar.vue [hunk 6]
```diff
@@ -202,6 +207,22 @@ export default {
 
     expandActionsAccordion() {
       this.SET_APP_SIDEBAR_EXPANDED_ACCORDION('files-actions')
+    },
+
+    onClickOutside(event) {
+      /*
+       * We need to go for this opt-out solution because under circumstances a model will be rendered,
```
Comment (pascalwengerter) [inline packages/web-app-files/src/components/Sidebar.vue:214]: Model = modal ?

Comment (pascalwengerter) [reply packages/web-app-files/src/components/Sidebar.vue:214]: Also in line below

**Pull request merged**